src/test/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/mapper/induction/InductionEntityMapperTest.kt

```diff
@@ -157,4 +160,39 @@ class InductionEntityMapperTest {
     verify(previousWorkExperiencesEntityMapper).fromEntityToDomain(inductionEntity.previousWorkExperiences!!)
     verify(workOnReleaseEntityMapper).fromEntityToDomain(inductionEntity.workOnRelease!!)
   }
+
+  @Test
+  fun `should update induction`() {
+    // Given
+    val inductionReference = UUID.randomUUID()
+    val prisonNumber = aValidPrisonNumber()
+    val existingInductionEntity = aValidInductionEntity(
+      reference = inductionReference,
+      prisonNumber = prisonNumber,
+    )
+    val updateInductionDto = aValidUpdateInductionDto(
+      reference = inductionReference,
+      prisonNumber = prisonNumber,
+      prisonId = "MDI",
+    )
+    val expectedInductionEntity = existingInductionEntity.deepCopy().apply {
+      id
+      reference = inductionReference
+      createdAtPrison = "BXI"
+      updatedAtPrison = "MDI"
```

Comment:
There's actually very little to update in the Induction entity itself - everything else is contained within associated classes, which I've added separate mappers and unit tests for. The main intention of this test is to make sure the relevant mappers are called correctly (i.e. in the `verify()` methods below).

Reply:
👍 
Given the intent (which may become lost/diluted after this PR?), any thoughts to naming the test something like
`should update induction by calling the correct child entity mappers` or similar?

Reply:
I'll change it to `should update induction and call related mappers`()